### DISCUSSION TITLE
More than one instance of Directus on one website with nuxt-directus module

### DISCUSSION THREAD

**Discussion by FreeSynergy** [directus/directus#16891] [Archive]
We have 2 different servers, each with a Directus account. Example news and events. On one website we want to make 2 queries, on both servers. We use nuxt 3 to build up the frontpage. We want to use the nuxt-directus plugin, to get the data from the 2 instances …

The website needs two querys:

    The news
    The events

Is it possible to do this with the nuxt-directus module???
If yes, how? (Where is the documentation?)

Of course there can be 2 different authorizations and so on …

**Comment by rijkvanzanten** [comment]
That's more a question for https://github.com/directus-community/nuxt-directus 🙂
